safe/plugin.py

```diff
@@ -252,6 +253,21 @@ def _create_minimum_needs_action(self):
         self.add_action(
             self.action_minimum_needs, add_to_toolbar=self.full_toolbar)
 
+    def _create_multi_buffer_action(self):
+        """Create action for multi buffer dialog."""
+        icon = resources_path('img', 'icons', 'show-minimum-needs.svg')
```

Comment:
Make a copy of an icon, like `show-minimum-needs.svg` if you want, and start using is in your code.